src/test/java/com/datadog/api/v1/client/api/LogsPipelinesLifecycleTest.java

```diff
@@ -68,15 +68,23 @@ public void pipelineLifecycleTest() throws ApiException {
         LogsProcessor logMessageRemapper = new LogsProcessor(new LogsMessageRemapper()
                 .addSourcesItem("source")
                 .name("log message remapper"));
-        LogsProcessor remapper = new LogsProcessor(new LogsAttributeRemapper()
+        LogsProcessor remapper1 = new LogsProcessor(new LogsAttributeRemapper()
                 .addSourcesItem("source")
                 .sourceType("tag")
                 .target("target")
                 .targetType("attribute")
                 .targetFormat(TargetFormatType.STRING)
                 .preserveSource(true)
                 .overrideOnConflict(true)
-                .name("log message remapper"));
+                .name("log message remapper1"));
+        LogsProcessor remapper2 = new LogsProcessor(new LogsAttributeRemapper()
+                .addSourcesItem("source")
+                .sourceType("tag")
+                .target("target")
+                .targetType("tag")
+                .preserveSource(true)
+                .overrideOnConflict(true)
+                .name("log message remapper2"));
```

Comment:
@vinckama can you please rename `remapper1` and `remapper2` variables to provide a reader more context?